src/v2/Apps/Auctions/Components/MyBids/MyBidsBidHeader.tsx

```diff
@@ -69,7 +72,7 @@ export const MyBidsBidHeaderFragmentContainer = createFragmentContainer(
     sale: graphql`
       fragment MyBidsBidHeader_sale on Sale {
         coverImage {
-          resized(width: 300, height: 100) {
+          cropped(width: 330, height: 100) {
```

Comment:
Was not aware of `cropped`. What this does is basically request an `objectFit: cover` version of the image from Gemini?

Reply:
Indeed